amethyst_renderer/src/pipe/pass.rs

```diff
@@ -41,7 +41,7 @@ impl Debug for Description {
 
 #[derive(Clone)]
 pub struct CompiledPass {
-    effect: Effect,
+    pub effect: Effect,
```

Comment:
doc: made field public without adding docs; I'm not sure yet if it is necessary to expose this.

Reply:
We have the deny_documentation lint turned on so if this is exposed then code won't compile.